src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -458,6 +459,12 @@ def load_arguments(self, _):
         c.argument('ip_version', min_api='2017-06-01', help='The IP version to update Microsoft Peering settings for.', arg_group='Microsoft Peering', arg_type=get_enum_type(['IPv4', 'IPv6']))
         c.argument('shared_key', help='Key for generating an MD5 for the BGP session.')
 
+    with self.argument_context('network express-route peering connection') as c:
+        c.argument('authorization_key', help='The authorization key.')
+        c.argument('address_prefix', help='/29 IP address space to carve out customer addresses for tunnels.')
+        c.argument('peering_name', options_list=['--peering-name'], help='Name of BGP peering.', id_part='child_name_1')
+        c.argument('connection_name', options_list=['--name', '-n'], help='Name of the peering connection.', id_part='child_name_2')
+        c.argument('peer_circuit', help='Name or ID of the peer ExpressRoute circuit.', validator=validate_er_peer_circuit)
```

Comment:
This is Resource ID of Private Peering for the peer circuit

Reply:
The CLI logic accepts just the name or ID of the circuit. It will automatically append '/peerings/{peering-name}' to form the correct ID the service needs. It will also accept the ID of the peering.